package.json

```diff
@@ -3,7 +3,24 @@
   "version": "0.0.0",
   "description": "Parser for Digital Extreme's profile data",
   "type": "module",
-  "exports": "./dist/*.js",
+  "exports": {
+    "./Parser": {
+      "types": "./dist/ProfileParser.d.ts",
+      "default": "./dist/ProfileParser.js"
+    },
+    "./Profile": {
+      "types": "./dist/Profile.d.ts",
+      "default": "./dist/Profile.js"
+    },
+    "./XpInfo": {
+      "types": "./dist/XpInfo.d.ts",
+      "default": "./dist/XpInfo.js"
+    },
+    "./Stats": {
+      "types": "./dist/Stats.d.ts",
+      "default": "./dist/Stats.js"
+    }
+  },
```

Comment:
typically the keys (`Parser`, `Profile`, etc.) should be lowercase, as they're part of the import, like `from '@wfcd/profile-parser/Parser` vs `@wfcd/profile-parser/parser`

also, are these esm or cjs exports? i haven't seen the actual export yet, and `default` will shove both at the same file, which usually ends badly?

Reply:
yes and I agree but warframe-status is using them with uppercase https://github.com/WFCD/warframe-status/blob/8e000c0a56724d678f369ae2883a9825bd58bfa1/src/controllers/profile.js#L3 and to keep it a passive update I choose to just keep them as so instead of breaking it for that project

Reply:
okiedoke, given that it was already esm, and the casing was already like this, i guess that's fine.